clients/pkg/promtail/scrapeconfig/scrapeconfig_test.go

```diff
@@ -30,14 +30,15 @@ pipeline_stages:
   - regex:
       expr: "./*"
   - json:
-      timestamp:
-        source: time
-        format: RFC3339
-      labels:
-        stream:
-          source: json_key_name.json_sub_key_name
-      output:
-        source: log
+      expressions:
+        timestamp:
+          source: time
+          format: RFC3339
+        labels:
+          stream:
+            source: json_key_name.json_sub_key_name
+        output:
+          source: log
```

Comment:
This might be my inexperience with Promtail but if this is supposed to following the [pipeline stage config](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#pipeline_stages), shouldn't it be more like this?
```suggestion
      expressions:
        timestamp: time
        stream: stream
        output: log
  - labels:
      stream:
  - timestamp:
      source: time
      format: RFC3339
  - output:
      source: output
```

The only documentation I can find regarding the original structure is in this historical design document: https://github.com/grafana/loki/blob/3039ca06f4d0fe964943bebcc29d9a4991143635/docs/sources/design-documents/labels.md?plain=1#L97-L104